EBA/Taschenrechner/LogFenster/Schule.cs

```diff
@@ -29,10 +29,10 @@ public  List<string> StartModule(Font font, Color foreColor, Color backColor)
             EingabeForm.BackColor = backColor;
             EingabeForm.ForeColor = foreColor;
             EingabeForm.AdviseParent += new EingabeForm.AdviseParentEventHandler(SetFromForm2);
-            ShowMessage("Bitte geben sie alle Noten an!(mit , Trennen)");
-            var noten = helper.Split(',');
-            Notenberechnung(noten);
-            Notenanzahl(noten);
+            ShowMessage("Bitte geben sie alle Noten an!(mit , Trennen)"); 
```

Comment:
put the `,` in a constant that is shared between this user message and the `.Split()` method in the next line. That will avoid incorrect error messages.